### PR TITLE
Upgrade Tailwind CSS v3.3

### PR DESCRIPTION
**Pull request by datlechin**
This PR is upgrade to Tailwind CSS v3.3.

What changes?
- Upgrade `@tailwindcss/forms`, `@tailwindcss/typography`, `prettier-plugin-tailwindcss` and `tailwindcss` to the latest version.
- Use css variable without the `var()` keyword
- Change `ml-*` to `ms-*, `mr-*` to `me-*`, ... to make RTL simpler. Read more about this (https://tailwindcss.com/blog/tailwindcss-v3-3#simplified-rtl-support-with-logical-properties)

### CODE REVIEW

File: packages/admin/resources/views/components/user-avatar.blade.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ])
 
 <div
-    {{ $attributes->class([
+ l   {{ $attributes->class([
```
Comment (zepfietje) [inline packages/admin/resources/views/components/user-avatar.blade.php]:
```suggestion
    {{ $attributes->class([
```

Comment (dat-archielite) [reply packages/admin/resources/views/components/user-avatar.blade.php]: thx :sweat_smile:

**Comment by zepfietje** [comment]
Thanks for the effort, @datlechin!
I really like these changes, though we probably can't merge this as it would be a breaking change.

What do you think, @danharrin?

**Comment by danharrin** [comment]
Yeah it would break v2 for people outside the admin panel or with custom themes. So v3 only 👍

Sorry about that

**Comment by datlechin** [comment]
@danharrin should it be created in `3.x` branch?

**Comment by danharrin** [comment]
If you want to do that work, go for it. But if you don't want to, we will do it before v3 release anyway :)

**Comment by mstfkhazaal** [comment]
> If you want to do that work, go for it. But if you don't want to, we will do it before v3 release anyway :)

I hope to add it as soon as possible for version 2

**Pull request closed without merging**